gemrb/core/Strings/StringView.h

```diff
@@ -101,6 +101,14 @@ class StringViewImp {
 		return data + len;
 	}
 
+	void clear() {
+		erase(0);
+	}
+	
+	void erase(size_type index) {
+		this->len = index;
+	}
+
```

Comment:
Why is this needed? `StringView` is almost never mutable.

Reply:
You suggested using `RTrim` in `RTrimCopy`, so we need to ducktype this.

Reply:
Huh, I pulled the code down and there doesnt seem to be any purpose for `RTrimCopy`.

```diff
-		auto key = RTrimCopy(SubStr(iniLine, keyStart, equalsPos - keyStart));
+		auto key = SubStr(iniLine, keyStart, equalsPos - keyStart);
+		RTrim(key);
```

Reply:
By now, yes. When I started it, I took other paths until here. 😉 We can leave it, I personally like chaining, or rewrite it again.